worlds/__init__.py

```diff
@@ -48,7 +49,7 @@ class WorldSource:
     path: str  # typically relative path from this module
     is_zip: bool = False
     relative: bool = True  # relative to regular world import folder
-    time_taken: Optional[float] = None
+    time_taken: float = -1.0
```

Comment:
This is an error I ran into with the changes I made.
`'<' not supported between instances of 'NoneType' and 'float'`
because this dataclass says that it's ordered.

Reply:
(The smuggled in was `typings/schema/__init__.pyi` - That has nothing to do with this PR.) 😛